AmplifyPlugins/API/AWSAPICategoryPlugin/APIError+Extension.swift

```diff
@@ -0,0 +1,33 @@
+//
```

Comment:
I generally avoid using `+Extension` in filename because the `Type+` pattern is supposed to described what's being added to that type through the extension (the `+` already implies extension).

So maybe a `APIError+Unauthorized.swift` is a better fit here?